### DISCUSSION TITLE
Regarding the size of RNN

### DISCUSSION THREAD

**Discussion by Yassawe** [seominjoon/kaist-ai605#85] [Q&A] [answered]
Helllo!

Are we free in choosing the number of spatial layers in RNN or should we stick to a single layer RNN? 

In other words, is the purpose of the exercises (3.2 and to an extent 4.1-2)  to compare MLP and RNN of similar complexity, or could we construct any arbitrarily large RNN?

thank you

**Comment by seominjoon** [comment, marked as answer]
The purpose of the exercise is more about comparing them architecturally, so you don't have to exactly match the parameter size, as long as the difference is not really huge.
